### PR TITLE
Fix v5 sdk errors and update swagger

### PR DESCRIPTION
**Pull request by ericshape**
This PR contains changes to address the v5 SDK validation errors. 
https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546
**### All the breaking changes are expected.** 

### 
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**### None of any of them. Add paths to address the validation errors which I listed at the beginning of the description.** 

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ericshape** [comment]
@jianyexi 
I don't fully understand this validation error:

❌ 1033 - RemovedProperty | The new version is missing a property found in the old version. Was 'tags' renamed or removed?New: Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L175:7Old: Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L178:7
-- | --


it seems there is no `tags` in the old one. Can you give me a hint about it? Thanks!

@xaliciayang @akning-ms

**Comment by jianyexi** [comment]
@ericshape  Actually it's inside the reference : https://github.com/Azure/azure-rest-api-specs/blob/fc163b10b549b83c8cfe20ecc4afefe417810c9d/specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L175

**Comment by ericshape** [comment]
> @ericshape Actually it's inside the reference :
> 
> https://github.com/Azure/azure-rest-api-specs/blob/fc163b10b549b83c8cfe20ecc4afefe417810c9d/specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L175

**Comment by ericshape** [comment]
❌ 1038 - AddedPath | The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/ManagedDatabaseTransparentDataEncryption.json#L160:5
-- | --
❌ 1038 - AddedPath | The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/TransparentDataEncryptions.json#L163:5



these breaking changes are expected 'cause the LIST operations are added to pass the validation in: 
https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546
❌ R4015 - NestedResourcesMustHaveListOperation

**Comment by akning-ms** [comment]
> Rule	Message
> ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/DataWarehouseUserActivities.json#L75:5
> ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/ManagedDatabaseTransparentDataEncryption.json#L160:5
> ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/TransparentDataEncryptions.json#L163:5
> these breaking changes are expected 'cause the LIST operations are added to pass the validation in:
> https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546
> ❌ R4015 - NestedResourcesMustHaveListOperation

Thanks @ericshape  + @jianyexi to help check list operation is correct to fix R4015.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json
Comment (bcham) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json:175]: Why are we switching this from a tracked resource to a proxy resource?

**Review by bcham — CHANGES_REQUESTED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json
Comment (ericshape) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json:175]: it was to fix the issue of 

❌ R3026 - TrackedResourcePatchOperation | Tracked resource 'RestorableDroppedDatabase' must have patch operation that at least supports the update of tags. It's strongly recommended that the PATCH operation supports update of all mutable properties as well.New: Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L167
-- | --

in https://msdata.visualstudio.com/Database%20Systems/_git/DsMainDev/pullrequest/508254

Comment (ericshape) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json:175]: this error was generated by https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546

**Comment by ericshape** [comment]
> > Rule	Message
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/DataWarehouseUserActivities.json#L75:5
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/ManagedDatabaseTransparentDataEncryption.json#L160:5
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/TransparentDataEncryptions.json#L163:5
> > these breaking changes are expected 'cause the LIST operations are added to pass the validation in:
> > https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546
> > ❌ R4015 - NestedResourcesMustHaveListOperation
> 
> Thanks @ericshape + @jianyexi to help check list operation is correct to fix R4015.

@jianyexi can you help to review this PR? Thanks!

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json
Comment (bcham) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json:175]: Are the owners of the RestorableDroppedDatabases resource type ok with this? This change is not without consequences...

Comment (ericshape) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json:175]: the change is from the owner at our request. just invited you to the group chat for more context.

**Review by bcham — APPROVED** [review]

**Review by jianyexi — APPROVED** [review]

**Comment by jianyexi** [comment]
> > Rule	Message
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/DataWarehouseUserActivities.json#L75:5
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/ManagedDatabaseTransparentDataEncryption.json#L160:5
> > ❌ 1038 - AddedPath	The new version is adding a path that was not found in the old version.New: Microsoft.Sql/preview/2020-08-01-preview/TransparentDataEncryptions.json#L163:5
> > these breaking changes are expected 'cause the LIST operations are added to pass the validation in:
> > https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546
> > ❌ R4015 - NestedResourcesMustHaveListOperation
> 
> Thanks @ericshape Ji Wang FTE + @jianyexi Jianye Xi FTE to help check list operation is correct to fix R4015.

**Comment by ericshape** [comment]
@filizt @jorgecotillo can you give ARM feedback? Thanks!
This PR addresses some lint errors in https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546. The breaking changes are expected and explained in the comment above.

**Comment by ericshape** [comment]
@pilor  I have sent an email to you for a review request about this PR

**Comment by pilor** [comment]
A PR flagged with breaking changes needs to go through the breaking change process outlined in the PR description.  Once you get approval there we can move forward.

**Comment by ericshape** [comment]
> A PR flagged with breaking changes needs to go through the breaking change process outlined in the PR description. Once you get approval there we can move forward.

Hi @pilor @akning-ms 
The breaking changes (adding new paths) were to address the validation errors in https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546. 
For this goal, do we also need to go through the long breaking change review process? https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes
It would cost a lot of overheads! Why did we need it?

We are pressured to release v5 SDK in https://github.com/Azure/azure-rest-api-specs/pull/12107 which is behind the schedule.

**Comment by pilor** [comment]
> > A PR flagged with breaking changes needs to go through the breaking change process outlined in the PR description. Once you get approval there we can move forward.
> 
> Hi @pilor @akning-ms
> The breaking changes (adding new paths) were to address the validation errors in https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546.
> For this goal, do we also need to go through the long breaking change review process? https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes
> It would cost a lot of overheads! Why did we need it?
> 
> We are pressured to release v5 SDK in #12107 which is behind the schedule.

You are making changes to an api-version that already exists in Microsoft.SQL's prod manifest.  That is exactly what the breaking change process is designed to cover.  This also shouldn't count as a preview API in my opinion as SQL has been using preview APIs as their main API versions since 2015.

**Review by pilor — APPROVED** [review]
Approving due to the breaking change s360 bypass

**Comment by ericshape** [comment]
> Approving due to the breaking change s360 bypass

@pilor Thank you very much!

**Pull request merged**
